docs/source/examples/private_utxo/overview.rst

```diff
@@ -0,0 +1,224 @@
+
+Holdings
+++++++++
+
+Holdings keep a record of which participants hold how much of which Assets.
+Holdings can be thought as on-Ledger wallet or bank account where the
+balance of each Asset is held.
+
+Unspent Transaction Output (UTXO)
++++++++++++++++++++++++++++++++++
+
+When off-Ledger, the assets are represented as a document. This document
+holds the details of the assets. The sha512 hash of this document is the UTXO
+handle. In this case, the document is in a very real sense the asset that is
+being represented. Having access to the document will provide the details of
+the asset and who owns it. As such it is important that the document is kept
+secret and protected.  UtxoDocument is considered valid when there is a
+matching UTXO record in the DLT. If there is not a matching entry in the DLT
+then the Asset has either been consumed or has not yet be recorded with an
+**Convert to UTXO** or **Transfer UTXO** transactions.
+
+
+.. literalinclude:: ../../../../families/private_utxo/protos/utxo_document.proto
+    :language: protobuf
+    :lines: 18-25
+
+Trading and Transaction's
+=========================
+
+Private UTXO Transaction Family
++++++++++++++++++++++++++++++++
+
+The Private UTXO Transaction Family is implemented by a Transaction Processor
+that manages representation in the Ledger.
+
+.. toctree::
+   :maxdepth: 2
+
+   private_utxo_transaction_family.rst
+
+
+Off-Ledger Trading
+++++++++++++++++++
+
+This section describes how off-Ledger Assets stored in UtxoDocuments
+are traded.
+
+UTXO processing Trusted Execution Environment (UTEE) are reponsible for
+validating and siging all off-Ledger transactions. UTEE's will be provided as
+signed binaries from the operators of the Ledger. The initial versions of the
+UTEE will support operation on clients enabled with SGX technology. The UTEE
+will utilize an IAS Proxy service to allow for generation of the Attestation
+Verification Reports(AVRs).
+
+The UTEE is responsible for enforcing the Asset semantic rules. This
+implementation only enforces the constant supply of Assets (Transfers neither
+create or destroy assets). Other semantics are possible but would require a
+different UTEE implementation to enforce those rules.
+
+The flow of an off-Ledger trade happens as follows.
+
+First, the UtxoDocument owner creates a set of Output documents that they wish
+to transform their document into. For example, if Alice has a UtxoDocument for
+100 units and wishes to give 10 to Bob. She would generate 2 output
+UtxoDocuments, 1 with Bob as the owner with the amount of 10 and the other with
+her as the owner and the amount of 90.
+
+Next, Alice will produce a signature of her input UtxoDocument using her
+private key. This signature acts as her authorization to consume the document.
+
+The Input UtxoDocument, the signature, and the output UtxoDocuments are all
+submitted to the UTEE for verification. When the verification finishes, a Quote
+is generated by the UTEE.
```

Comment:
Quote and Attestation are not synonymous. Quote is an assertion by the Enclave. Assestation is a signed valid quote by the Attestation Service.